.github/scripts/cleanup-cluster.sh

```diff
@@ -21,22 +21,26 @@ set -e
 echo "cleaning up cluster"
 
 # Delete all test resources in queuemessages.
-echo "delete all resources in queuemessages.ucp.dev"
-kubectl delete queuemessages.ucp.dev -n radius-system --all
+if kubectl get crd queuemessages.ucp.dev > /dev/null 2>&1; then
+    echo "delete all resources in queuemessages.ucp.dev"
+    kubectl delete queuemessages.ucp.dev -n radius-system --all
+fi
 
 # Testing deletion of deployment.apps.
```

Comment:
Is this comment valid or was it added for some testing? I know it is unrelated to the changes in the PR but I just came across it so thought to ask

Reply:
I added this and forgot to remove.